crates/chia-protocol/src/header_block.rs

```diff
@@ -72,6 +73,18 @@ impl HeaderBlock {
     pub fn first_in_sub_slot(&self) -> bool {
         !self.finished_sub_slots.is_empty()
     }
+
+    pub fn into_unfinished_header_block(self) -> UnfinishedHeaderBlock {
```

Comment:
I don't think there's a counterpart to this in `chia-blockchain`, is there?

Reply:
It's to do this, this helper function isn't needed in the Python code, but it makes sense for the Rust port imo:
https://github.com/Chia-Network/chia-blockchain/blob/main/chia/consensus/block_header_validation.py#L842

Reply:
That's why I didn't write a binding for it